src/Kernel32.Desktop/Kernel32+JobObjectInfoClass.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="JOBOBJECT_INFO_CLASS"/> nested type.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// The information class for the limits to be set.
+        /// </summary>
+        [Flags]
```

Comment:
Are these really flags? The ordinal values assigned to the enum values don't look like they're flags. And I had difficulty finding the MSDN page that documents these constants. Can you please add a link to it in a `<remarks>` section of the doc for this type?